### ISSUE TITLE
Preview glitch on bot description

### ISSUE THREAD

**Issue by BobDotCom** [top-gg/issues#448]
#### Expected Behavior

I expected a preview of my description

#### Observed Behavior

Button didn't do anything, console says `ReferenceError: Can't find variable: preview`

#### Applicable Links or Attachments

![What it looks like](https://media.discordapp.net/attachments/748937725079453746/772594276487790603/Screen_Shot_2020-11-01_at_4.52.47_PM.png?width=927&height=522)

#### Steps to Reproduce

1. Open top.gg
2. Make a new bot
3. Click preview on the bot description

#### Web Browser and Version

Occurs on both:
Safari Version 14.0 (15610.1.28.1.9, 15610)
Google Chrome Version 86.0.4240.75 (Official Build) (x86_64)

#### Operating System and Version

macOS Catalina Version 10.15.7 (19H2)

#### Screen Resolution

1440 x 900

#### Additional Information/Comments

[Ur logo went squish](https://media.discordapp.net/attachments/748937725079453746/772595283841646653/Screen_Shot_2020-11-01_at_4.57.27_PM.png?width=898&height=523)

**Comment by ExtremeDevelopments** [comment]
I was able to recreate the bug on chrome, 
![image](https://user-images.githubusercontent.com/48730677/97818208-d0c8e200-1c66-11eb-8104-6582404fbbde.png)

Although, I was not able to recreate the logo stretch, I can confirm that this bug does exist for the **Chrome** browser.

**Comment by Xetera** [comment]
You can't preview html for a bot that hasn't been added since it doesn't have an existing page to show to the user. That button will just be removed instead.

**Comment by BobDotCom** [comment]
That makes sense. Will it work for editing a description once it is made?

**Comment by Xetera** [comment]
Yes, it's just the initial submission that doesn't have preview capability. Feel free to create another issue if you have problems previewing changes for an existing bot page

**Comment by ExtremeDevelopments** [comment]
Ah that makes more sense was wondering why I was only getting on a new bot and not on my existing bot
